### ISSUE TITLE
Hubot returns undefined

### ISSUE THREAD

**Issue by purplefish32** [LoicMahieu/hubot-commitstrip#1]
Hubot returns undefined when asked : "hubot commitstrip"

**Comment by LoicMahieu** [comment]
Yes, fetching commitstrip is buggy at this time. I didn't really investigate into this. If you have a fix, send a PR :)
